simbad/util/mtz_util.py

```diff
@@ -408,6 +410,46 @@ def crystal_data(mtz_file):
     return space_group, resolution, cell_parameters
 
 
+def change_space_group(input_mtz, output_mtz, new_space_group):
```

Comment:
I think this function should be incorporated in the class above. We have all the I/O stuff there anyway so a simple property should do. I would be much cleaner IMO.

Reply:
I've made these changes